Description: Pass the DEBIAN_LDFLAGS environment variable to the linker
Origin: vendor
Author: Noah Meyerhans <noahm@debian.org>
Index: iputils/Makefile
===================================================================
--- iputils.orig/Makefile
+++ iputils/Makefile
@@ -136,7 +136,7 @@ all: $(TARGETS)
 	$(COMPILE.c) $< $(DEF_$(patsubst %.o,%,$@)) -o $@
 LINK.o += $(CFLAGS)
 $(TARGETS): %: %.o
-	$(LINK.o) $^ $(LIB_$@) $(LDLIBS) -o $@
+	$(LINK.o) $^ $(DEBIAN_LDFLAGS) $(LIB_$@) $(LDLIBS) -o $@
 
 # -------------------------------------
 # arping
